docs/get-started.rst

```diff
@@ -8,9 +8,19 @@ lab earth analytics courses.
 Earthpy Modules
 ---------------
 
-Spatial
-IO
-Utils
+- Spatial: Various functions for loading, manipulating, and displaying raster data
+  - `create_raster_stack`: generates a single multi-band raster from separate raster TIFs,
```

Comment:
Did you intent this to be `create_raster_stack` using code highlighting?

Reply:
Sorry. I'd thought that having input from you and Max on what the changes should include was my go ahead. Lesson learned.